### PR TITLE
fix: add sqlness tests for some promql function

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add sqlness tests for 
- stddev
- stdvar
- quantile_over_time

Other test are not ported due to 
- We don't support to insert NaN/Inf via sql
- Some tests will be expanded to thousands of lines. I'd like to wait https://github.com/CeresDB/sqlness/issues/51

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: tests/cases/standalone/common/tql/aggr_over_time.sql [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    (20000, 2, "three samples"),
+    (0, 0, "uneven samples"),
+    (10000, 1, "uneven samples"),
+    (20000, 4, "uneven samples");
+
+-- eval instant at 1m quantile_over_time(0, data[1m])
+-- 	{test="two samples"} 0
+-- 	{test="three samples"} 0
+-- 	{test="uneven samples"} 0
+-- tql eval (60, 60, '1s') quantile_over_time(0, data[1m]);
```
Comment (waynexia) [inline tests/cases/standalone/common/tql/aggr_over_time.sql:75]: result set is unstable, waiting for https://github.com/CeresDB/sqlness/issues/57

File: src/promql/src/planner.rs [hunk 2]
```diff
@@ -885,8 +886,8 @@ impl PromPlanner {
                 ScalarFunc::Udf(PredictLinear::scalar_udf(t_expr))
             }
             "holt_winters" => {
-                let sf_exp = match other_input_exprs.get(0) {
-                    Some(DfExpr::Literal(ScalarValue::Float64(Some(sf)))) => *sf,
+                let sf_exp = match other_input_exprs.pop_front() {
```
Comment (etolbakov) [inline src/promql/src/planner.rs:889]: Thanks for improving it. Looks much better than `.get(0)` 👍

**Review by etolbakov — APPROVED** [review]
LGTM

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**